pytest-selenium/pages/content.py

```diff
@@ -36,6 +51,31 @@ def sidebar(self):
     def attribution(self):
         return self.Attribution(self)
 
+    @property
+    def section_url_within_attribution(self):
```

Comment:
is this unused?

Reply:
@TomWoodward this is used in test_section_url_in_citation_text_shows_url_for_current_page